### ISSUE TITLE
Make `value` argument of `/voucher` entry-point optional

### ISSUE THREAD

**Issue by guidanoli** [cartesi/openapi-interfaces#21]
## Rationale

On SDK v1, vouchers had only two fields: `destination` and `payload`.
On-chain, this triggers a [`CALL`](https://www.evm.codes/?fork=cancun#f1) instruction.
This instruction also takes a `value` as stack input, but we don't specify it.
Solidity passes zero as the default value to message calls.

On SDK v2, we introduced the `value` field to parameterize this stack input.
In most cases, however, this field will still be zero.
It will only be non-zero when there is some movement of Ether.
The classic example is Ether withdrawals.

If this field must be explicitly passed through the HTTP API,
application back-ends will have to adapt every call to the `/voucher` entry-point,
just to pass the same value, zero.
We believe this to be a sensible default,
because it is the current value,
and because most vouchers will not involve any Ether transfer.

## Proposal

Make the `value` argument of the `/voucher` entry-point optional.
The default value should be zero, just like before.
If this change is agreed upon, an equivalent issue should be opened on [`machine-emulator-tools`](https://github.com/cartesi/machine-emulator-tools).

**Comment by jplgarcia** [comment]
Strong agree, but I would like to point that we also have to change the description of destination 
https://github.com/cartesi/openapi-interfaces/blob/9b9d6088d6e2f93c333d5500e868116d7e57ba91/rollup.yaml#L406
```          
description: 20-byte address of the destination contract for which the payload will be sent.
```
Currently it specifiec it is the address of a contract and for ether withdrawals it can be the user who will be receiving the assets. 
Correct @guidanoli ?

**Comment by guidanoli** [comment]
Yes, `destination` can be any Ethereum account address.
If the account has some code, we say it's a smart contract.

**Comment by guidanoli** [comment]
> It’s basically removal of line
> https://github.com/cartesi/openapi-interfaces/blob/9b9d6088d6e2f93c333d5500e868116d7e57ba91/rollup.yaml#L429

Yes, and we should make it clear that zero is the default value.

**Comment by miltonjonat** [comment]
Hi guys. Any reason why this didn't go forward?

**Comment by guidanoli** [comment]
@vfusco
